packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/web3/utils.ts

```diff
@@ -0,0 +1,136 @@
+import BigNumber from 'bignumber.js'
+import { getExponentValue } from '@masknet/web3-shared'
+
+export function getHumanValue(value?: BigNumber, decimals = 0): BigNumber | undefined {
+    return value?.div(getExponentValue(decimals))
```

Comment:
why not use `pow10`

Reply:
I grabbed this from the BarnBridge main app => most of this isn't really needed. Let me clean this up/delete